CMakeLists.txt

```diff
@@ -130,6 +130,16 @@ if (NOT BLT_LOADED)
   include(${BLT_SOURCE_DIR}/SetupBLT.cmake)
 endif()
 
+#######################################
+# Options that depend on BLT Options
+#######################################
+cmake_dependent_option( UMPIRE_ENABLE_TESTS "Build Umpire tests" On
+                       "ENABLE_TESTS" Off )
+cmake_dependent_option( UMPIRE_ENABLE_DOCS "Build Umpire docs" On
+                        "ENABLE_DOCS" Off )
+cmake_dependent_option( UMPIRE_ENABLE_EXAMPLES "Build Umpire examples" On
+                        "ENABLE_EXAMPLES" Off )
+
```

Comment:
@davidbeckingsale -- I applied the same fix as in RAJA here.  Sorry I didn't notice that in my setup since BLT was loaded in the parent project. Hopefully, this should be all good now.